doc/external-gluster-storage.md

```diff
@@ -1,12 +1,59 @@
 # Use external gluster volume as PV in kadalu
 
 If you are having your storage nodes outside of kubernetes cluster, you
-can still use kadalu.
+can still use kadalu. We call the `kind` of this setup as 'External' mainly
+because, the storage server processes are not managed by kadalu operator
+in this case.
 
-All you need is to have a sample like [`examples/sample-external-storage.yaml`](../examples/sample-external-storage.yaml).
+## Using external gluster in kadalu native way
 
-Please edit 'gluster_host' and 'gluster_volname' parameters in Storage Class.
+In this mode, we expect gluster volume to be created and is in 'Started' state.
+kadalu storage config takes one of the node IP/hostname, and gluster volume name
+to use it as the storage for PVs. The PVs would be provided as subdirectories,
+similar to how a PV is created in kadalu native way.
+
+The best example for this is, what we use in our CI. Checkout this
+[sample yaml file](../examples/sample-external-kadalu-storage.yaml).
+
+Please edit 'gluster_host' and 'gluster_volname' parameters in config section.
+
+You can also setup the external storage through our `kubectl kadalu` CLI.
+
+```console
+kubectl kadalu storage-add store-name --external gluster.kadalu.io:/kadalu
```

Comment:
`--type External`

Reply:
As it was explicitly set, I was thinking of avoiding it in docs.